test/e2e/multicast_test.go

```diff
@@ -58,19 +57,13 @@ func TestMulticast(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error computing multicast interfaces: %v", err)
 	}
-	t.Run("testMulticastWithNoEncap", func(t *testing.T) {
-		skipIfEncapModeIsNot(t, data, config.TrafficEncapModeNoEncap)
-		runMulticastTestCases(t, data, nodeMulticastInterfaces, true)
-	})
-	t.Run("testMulticastWithEncap", func(t *testing.T) {
-		ac := func(config *agentconfig.AgentConfig) {
-			config.TrafficEncapMode = "encap"
-		}
-		if err := data.mutateAntreaConfigMap(nil, ac, true, true); err != nil {
-			t.Fatalf("Failed to deploy cluster with encap mode: %v", err)
-		}
-		runMulticastTestCases(t, data, nodeMulticastInterfaces, false)
-	})
+	encapMode, err := data.GetEncapMode()
+	if err != nil {
+		t.Fatalf("Failed to get encap mode: %v", err)
+	}
+	// Only check receiver router in encap mode.
+	checkReceiverRoute := encapMode == config.TrafficEncapModeNoEncap
```

Comment:
is there a mismatch between the comment and the code? Should the comment read "in noEncap mode"